### ISSUE TITLE
Add support for Innova 2.0 ELEC

### ISSUE THREAD

**Issue by jeffb1974** [danielrivard/homeassistant-innova#203]
**Is your feature request related to a problem? Please describe.**
Looks like Innova has released new versions of 2.0 AC units, e.g. I just added a [2.0 ELEC](https://www.innovaenergie.com/en/products/air-conditioning-without-outdoor-unit/2.0-verticale/2.0-elec/).  They have R32 gas but seem to have some differences when adding to Wifi and configuring in the Innova App. Seems there is a new App just for the newer 2.0 units.

When trying to add to HA via the Innova HACS 'add entry' it does not connect, with the message Failed to Connect.

**Describe the solution you'd like**
Enable the onboarding and control of the new Innova 2.0 units in the same way as the older version(s).  I have no idea if this is a simple request or requires a totally new integration, but wanted to raise it as a feature request regardless.

Are there is a way to get you logs of any other information that will help to understand if this is user error, a defect or an enhancement request?

Here is the log file when attempting to add the device:
This error originated from a custom integration.

Logger: innova_controls.network_functions
Source: custom_components/innova/config_flow.py:37
integration: Innova (documentation, issues)
First occurred: 16:45:40 (8 occurrences)
Last logged: 17:38:09

Bad logger message: Error getting status http://172.24.1.201/api/v/1/status ((ClientConnectorError(ConnectionKey(host='172.24.1.201', port=80, is_ssl=False, ssl=True, proxy=None, proxy_auth=None, proxy_headers_hash=1412496468314771007), ConnectionRefusedError(111, "Connect call failed ('172.24.1.201', 80)")),))
Bad logger message: Error getting status http://OfficeAC/api/v/1/status ((ClientConnectorError(ConnectionKey(host='officeac', port=80, is_ssl=False, ssl=True, proxy=None, proxy_auth=None, proxy_headers_hash=1412496468314771007), OSError('DNS server returned answer with no data')),))
Bad logger message: Error getting status http://172.24.1.200/api/v/1/status ((ServerDisconnectedError('Server disconnected'),))

**Comment by danielrivard** [comment]
Yes, someone reported that a few weeks ago. https://github.com/danielrivard/homeassistant-innova/discussions/194

Unfortunately it seems like they have completely changed the API and protocol, so this integration for now is not compatible with the newer units and their firmware.

Since I only own one unit and it's an older one (and I don't plan to upgrade it), it would be hard for me to add support for it. Unless they eventually upgrade the firmware on older units.

But I feel like it is so different that it should maybe be a different integrations altogether.

**Comment by MidasLamb** [comment]
@danielrivard , I might be buying an Innova, so it'll probably be a newer model. Do you mind sharing how you went about discovering the protocol to create this integration? If I buy it then, I can look into what the new api/protocol is.

I assume you did some packet capturing?

**Comment by danielrivard** [comment]
@MidasLamb the previous version had a "desktop application" which was mostly a local webapp (https://www.innovaenergie.com/site/assets/files/2794/twopointzero.zip) from which I was able to trace a lot of the network calls.

I also decompiled the Android app to find more missing calls.

I did look at their new Android app, but the protocol seems a bit more complex, and also requires authentication with an email and password, which should be possible to do in homeassistant, but it becomes a bit more sensitive to handle.

**Comment by danielrivard** [comment]
Take a look at this integration that another developer built for the new API, it is worth a try. Let me know if you have success with your unit.

https://github.com/ChristophHohner/homeassistant-innova-duepuntozero

**Comment by achillecalegari** [comment]
For the newer 2.0 units that only work with the Solution Tech "Innova" app (cloud gRPC, no local REST API on port 80): I have published a separate integration that supports them, see https://github.com/achillecalegari/hass-innova-cloud (details and protocol notes in #278). It uses its own domain, so it can coexist with this one for the older units.
